src/minErr.js

```diff
@@ -36,17 +36,27 @@ function minErr(module, ErrorConstructor) {
     var SKIP_INDEXES = 2;
 
     var templateArgs = arguments,
+      argsDebuggedStringIndexes = [],
       code = templateArgs[0],
       message = '[' + (module ? module + ':' : '') + code + '] ',
       template = templateArgs[1],
       paramPrefix, i;
 
+    function getTemplateArgAndConvertToDebugStringFormat(index) {
```

Comment:
Oh, I see. It doesn't matter much, because most of the time the arguments will be strings (which are really cheap to call `toDebugString()` on), but I agree there is unnecessary overhead.
Yet your implementation seems too complicated (especially given that we are going to use every argument at least once), how about we simplify it like this:

```
-    var templateArgs = arguments,
-        code = templateArgs[0],
-        message = '[' + (module ? module + ':' : '') + code + '] ',
-        template = templateArgs[1],
+    var code = arguments[0],
+        message = '[' + (module ? module + ':' : '') + code + '] ',
+        template = arguments[1],
+        templateArgs = sliceArgs(arguments).map(function(arg) { return toDebugString(arg, objectMaxDepthInErrorMessage); }),
```